api/db/migrate/20221221122100_add_secret_key_to_spree_webhooks_subscribers.rb

```diff
@@ -0,0 +1,5 @@
+class AddSecretKeyToSpreeWebhooksSubscribers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :spree_webhooks_subscribers, :secret_key, :string, null: true
```

Comment:
I think `OpenSSL::HMAC.digest('sha256', @event.subscriber.secret_key, payload)` will fail when the key parameter is `nil`. I'd suggest adding two more migrations as a part of this PR:
1. A migration that will generate and backfill empty secret_keys for spree_webhooks_subscribers
2. A migration that will disallow nulls in the secret_key column

This way we'll make sure we won't break any webhook subscribers in existing installations of Spree

Reply:
Thank you! I'll add these migrations. As I said before, I wasn't fully aware of how the Spree core team likes to handle these kinds of migrations. I'll make the adjustments.

Reply:
Thanks!